online_meetings/bluejeans_report.py

```diff
@@ -0,0 +1,186 @@
+import requests
+# standard libraries
+import json
+import logging
+import os
+import sys
+import yaml
+
+# third-party libraries
+import pandas as pd
+import furl
+import time
+import dateparser
+import pytz
+from datetime import datetime, timedelta
+import glob
+
+# importing required modules
+from zipfile import ZipFile
+
+# Initialize settings and globals
+
+logger = logging.getLogger(__name__)
+
+# Default time to sleep when hitting a 429 limit
+DEFAULT_SLEEP_TIME = 10
+
+# read configurations
+try:
+    with open(os.path.join(os.path.dirname(__file__), '../config/env.json')) as env_file:
+        ENV = yaml.safe_load(env_file.read())
+except FileNotFoundError:
+    sys.exit(
+        'Configuration file could not be found; please add env.json to the config directory.')
+
+logging.basicConfig(level=ENV.get('LOG_LEVEL', 'DEBUG'))
+
+url = "https://api.bluejeans.com/oauth2/token?Client"
+
+payload = "{ \"grant_type\": \"client_credentials\", \"client_id\": \"Metrics_BJN\", \"client_secret\": \"" + ENV.get('BLUEJEANS_CLIENT_SECRET') + "\"}"
+logger.info(payload)
+headers = {
+  'Accept': 'application/json',
+  'Content-Type': 'application/json'
+}
+
+token_expiration_time = datetime.now()
+response = requests.request("POST", url, headers=headers, data=payload).text.encode('utf8')
+token_json = json.loads(response)
+token = token_json["access_token"]
+logger.info(token)
+## the expiration time in seconds
+expires_in_sec = token_json["expires_in"]
+## minus for one minute, for safty check
+expires_in_sec = expires_in_sec - 60
+token_expiration_time = datetime.now() + timedelta(seconds=expires_in_sec)
+logger.info(f"{token} expires on: {token_expiration_time}")
+
+# construct param string BLUEJEANS_MEETINGS_PARAMS
+
+params = ENV["BLUEJEANS_MEETINGS_PARAMS"]
```

Comment:
I don't see what you're using either of these variables `f` or `url` for. I removed furl from the other method as it wasn't used.